src/main/resources/plugin.yml

```
@@ -1,6 +1,6 @@
 main: org.skriptlang.addonpatcher.AddonPatcher
 name: AddonPatcher
-version: 1.0-SNAPSHOT
+version: patcher-2.7.0-dev1
```

```suggestion
version: 2.7.0-dev1
```

There's not really a need to have 'patcher' in the version of the AddonPatcher